scripts/inventory.js

```diff
@@ -164,7 +164,26 @@ const filters = [
   },
 ];
 
-async function loadInventory() {
+/**
+ * Loads inventory data from the server.
+ * @returns {Promise<Array>} The inventory data.
+ * @throws {Error} If the fetch request fails.
+ */
+async function loadInventoryData() {
+  const response = await fetch('/data/hubblehomes.json?sheet=inventory');
```

Comment:
Can we check to see if have already loaded the inventory data?  The createCommunityInventoryMap provides us with a Map stored in the global object `hh.inventory`.  Or, we cache the inventory raw in hh.inventory, and then createCommunityInventoryMap returns the map version using the hh.inventory.  

I think we should make this change.

Reply:
I have updated the code, please take a look.